src/main/resources/ui.properties

```diff
@@ -136,3 +136,8 @@ updateCancel = Quit
 updateStart = Update
 updateRequired = Your version is too outdated. An update is required.
 updatePossible = An update is available. You can now update if you wish to.
+
+desktopAppVersion = Desktop App
```

Comment:
The official name is `Qabel Desktop Client`.
And the name of that property suggests a version, that is missing. You could add it as a placeholder